src/cloud/internal/cloudservice.h

```diff
@@ -76,8 +88,10 @@ private slots:
     network::RequestHeaders headers() const;
 
     RequestStatus downloadUserInfo();
+    RequestStatus doUploadScore(system::IODevice& scoreSourceDevice, const QString& title, const QUrl& sourceUrl = QUrl());
```

Comment:
It is strange that we have `IODevice` in the system namespace, not io.

Reply:
moved IODevice into mu::io